### PR TITLE
feat(NFDIV-509): You can use English or Welsh courts interstitial page

### PR DESCRIPTION
**Pull request by SebFlippence**
### JIRA link ###

[NFDIV-509](∫https://tools.hmcts.net/jira/browse/NFDIV-509) Content for "You can use courts of England & Wales" interstitial page

### Change description ###

Adds content and updated feature tests for "You can use English or Welsh courts" page.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/common/page.njk [hunk 1]
```diff
@@ -8,10 +8,12 @@
   <!-- {{ formState.id }} -->
 {% endblock %}
 
+{% set serviceName = (applyForDivorce if isDivorce else applyForDissolution) | capitalize %}
```
Comment (linusnorton) [inline src/main/steps/common/page.njk:11]: Shouldn't this just be a variable set in the common content?

File: src/main/steps/sequence.ts [hunk 2]
```diff
@@ -166,7 +166,10 @@ export const sequence: Step[] = [
           return JURISDICTION_INTERSTITIAL_URL;
 
         case `${YES}${NO}`:
-          return LIVING_ENGLAND_WALES_SIX_MONTHS;
+          if (data.yourLifeBasedInEnglandAndWales === YES) {
```
Comment (linusnorton) [inline src/main/steps/sequence.ts:169]: ternary operator please

**Review by linusnorton — APPROVED** [review]
Couple of thoughts but looks good

**Review by SarahLittlejohn — APPROVED** [review]

File: src/main/steps/common/page.njk (hunk 1 above)
Comment (SebFlippence) [reply src/main/steps/common/page.njk]: 👍 Good idea, I've moved this out to common content in 6f7690a

File: src/main/steps/sequence.ts (hunk 2 above)
Comment (SebFlippence) [reply src/main/steps/sequence.ts]: 👍 Good spot, updated in 901b646

**Pull request merged**